remodel/models.py

```diff
@@ -24,7 +24,8 @@ def __new__(mcs, name, bases, dct):
             return super_new(mcs, name, bases, dct)
 
         # Set metadata
-        dct['_table'] = tableize(name)
+        if '_table' not in dct:
```

Comment:
I'm thinking, since specifying a table is "public" now, I would have `dct['table']` rather than `dct['_table']`. This would lead to a more readable API as well:

```python
class Artist(Model):
    table = 'artist_tbl'
```